### DISCUSSION TITLE
Add a song delay bar

### DISCUSSION THREAD

**Discussion by RobotoSkunk** [ppy/osu#23898] [Ideas] [answered]
I have the idea to add a song delay bar, to synchronize the song with the actual gameplay for low-end computers or bluetooth devices.

**Comment by peppy** [comment, marked as answer]
This is already available at the loading screen, look for audio offset.

If you want to apply a delay/offset for all beatmaps, adjust the universal offset in settings instead.

**Comment by RobotoSkunk** [comment]
Thank you! I've thought that it was called "delay"

**Reply by peppy** [reply]
I've added "delay" as a search term for offset settings in https://github.com/ppy/osu/pull/23899, thanks for the tip.
